packages/analyticsdx-template-lint/src/schemas/layout-schema.json

```diff
@@ -626,6 +627,173 @@
         }
       },
       "required": ["type", "image"]
+    },
+    "guidancePanel": {
+      "type": "object",
+      "description": "The guidance panel specification for the page.",
+      "additionalProperties": false,
+      "required": ["title"],
+      "properties": {
+        "title": {
+          "type": "string",
+          "description": "Guidance panel title",
+          "doNotSuggest": false
+        },
+        "backgroundImage": {
```

Comment:
Should we refactor this to its own type under references? (Its the same as the `backgroundImage` in a page)